lib/vintage_net_wizard/web/endpoint.ex

```diff
@@ -69,26 +77,72 @@ defmodule VintageNetWizard.Web.Endpoint do
     ]
   end
 
+  defp maybe_with_redirect([main, redirector]) do
```

Comment:
This is a nit, but can we remove the logic from specifying children in `maybe_use_ssl` and put it here? Just do a lookup of the port and if its not 80, start the redirect